library/src/main/java/com/pokegoapi/api/map/pokemon/CatchResult.java

```diff
@@ -93,4 +93,9 @@ public boolean isFailed() {
 		}
 		return (this.getStatus() != CatchStatus.CATCH_SUCCESS || failed);
 	}
+
+	public boolean hasCaptureAward()
+	{
```

Comment:
That's a perfectly valid java style, lol... But yeah, inconsistent with your project and the java std, sorry

El dia 04/08/2016 21:34, Michał Górski <notifications@github.com> va escriure:

In library/src/main/java/com/pokegoapi/api/map/pokemon/CatchResult.java<https://github.com/Grover-c13/PokeGOAPI-Java/pull/482#discussion_r73588137>:

> @@ -93,4 +93,9 @@ public boolean isFailed() {
>               }
>               return (this.getStatus() != CatchStatus.CATCH_SUCCESS || failed);
>       }
> +
> +     public boolean hasCaptureAward()
> +     {


please format it to Java style :D

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/Grover-c13/PokeGOAPI-Java/pull/482/files/57d5929e961a9333534ef2b41d5f00e80962c436#r73588137>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AGKpr0ka7mizGL44-q-fpTyX12ONuzWIks5qcj7DgaJpZM4JcN_f>.